packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -80,6 +80,12 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
     required this.fadeoutOpacityAnimation,
     Color trackColor = const Color(0x00000000),
     Color trackBorderColor = const Color(0x00000000),
+    bool showButtons = false,
```

Comment:
Could this be removed if we just consider `buttonsShape == null` means don't show buttons?